.codespell/.codespellrc

```diff
@@ -1,5 +1,5 @@
 [codespell]
 quiet-level = 2
 count =
-skip = ./deps,./src/crc16_slottable.h,tmp*,./.git,./lcov-html
+skip = ./deps,./src/crc16_slottable.h,tmp*,./.git,./lcov-html,./src/rax-test.c
```

Comment:
instead of skipping it, why not just fix the spelling issues it complains about?